tony-history-server/app/controllers/JobsMetadataPageController.java

```diff
@@ -0,0 +1,43 @@
+package controllers;
+
+import com.typesafe.config.Config;
+import java.util.ArrayList;
+import java.util.List;
+import javax.inject.Inject;
+import models.JobMetadata;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import play.Logger;
+import play.Logger.ALogger;
+import play.mvc.Controller;
+import play.mvc.Result;
+
+import static utils.HdfsUtils.*;
+import static utils.ParserUtils.*;
+import static utils.SecurityUtils.*;
+
+
+public class JobsMetadataPageController extends Controller {
+  private static final ALogger LOG = Logger.of(JobsMetadataPageController.class);
+  private final Config config;
+
+  @Inject
+  public JobsMetadataPageController(Config config) {
+    this.config = config;
+    getInstance(config);
+  }
+
+  public Result index() {
+    FileSystem myFs = getFs();
+    LOG.info("Successfully instantiated file system");
```

Comment:
This LOG message probably doesn't need to be at the `info` level. I suggest removing it or changing it to `debug`.

Reply:
Removed: https://github.com/linkedin/TonY/pull/62/commits/26c4ec1f972a5c8006ec1744ade806787e940265